testes/index.spec.js

```diff
@@ -0,0 +1,7 @@
+const imported = require("../src/status")
+
+describe("Testing my function imported", () => {
+  it("Return an object with a boolean value atribute", () => {
+    expect(imported.run()).toStrictEqual({running: expect.any(Boolean)})
```

Comment:
Why `expect.any(Boolean)`? You should expect a specific  boolean value, shouldn't you?

Reply:
expect true... my mistake! I'll fix this.

Reply:
The testes/index.spec.js is testing src/index.js! How can i fix my mistake? I was search for a exemple and i am confused. Should i need rename my repositorio just my file 'spec' is in?